components/samples/a11y/config.js

```diff
@@ -0,0 +1,50 @@
+MathJax = {
+    loader: {
+        load: ["a11y/complexity", "tex-input"],
+        paths: {
+            mathjax: '../../dist'
+        },
+        dependencies: {
+            'a11y/semantic-enrich': ['core', 'sre', 'mml-input'],
+            'a11y/complexity': ['core', 'sre', 'mml-input']
+        },
+        source: {
+            sre: '../../node_modules/speech-rule-engine/lib/sre_browser.js'
+        }
+    },
+    startup: {
+        input: ['tex'],
+        ready() {
+            MathJax.startup.defaultReady();
+//            console.log(MathJax.chtmlStylesheet());
+            console.log(MathJax.tex2mml('a+b+c+d+e'));
+//            console.log(MathJax.startup.adaptor.outerHTML(MathJax.tex2chtml('x+1')));
+//            MathJax.tex2mmlPromise('x+1').then((mml) => console.log(mml));
+//            MathJax.Typeset();
+//            const adaptor = MathJax.startup.adaptor;
+//            console.log(adaptor.outerHTML(adaptor.body(MathJax.startup.document.document)));
```

Comment:
Remove.

Reply:
There are there for testing for me.  I'll remove them before the final version.  I'm not sure these samples will be kept in the distribution in the end at all.